x-pack/plugins/asset_inventory/server/lib/get_k8s_cluster.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SearchRequest } from '@elastic/elasticsearch/lib/api/types';
+import { debug } from '../../common/debug_log';
+import { EcsOrchestratorFieldset, K8sCluster } from '../../common/types_api';
+import { ASSETS_INDEX } from '../constants';
+import { esClient } from './es_client';
+import { getK8sNodes } from './get_k8s_nodes';
+
+export async function getK8sCluster(name: string): Promise<K8sCluster> {
+  const dsl: SearchRequest = {
+    index: ASSETS_INDEX,
+    query: {
+      bool: {
+        must: [
+          {
+            term: {
+              ['asset.name.keyword']: name,
```

Comment:
Why do you need `.keyword` here and below? Is this because the template is missing? Lets make sure we always have the template in place and then have the proper queries. Same for everything below.

Reply:
Yeah, I'll reintegrate the mappings with the flow of using all of those scripts and adjust these queries.

Reply:
Done. https://github.com/jasonrhodes/kibana/pull/14/commits/dd2eddcfb1d5d0f2ab08c4c66a32f580c67d0128